plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -861,7 +866,8 @@ public void createMaterializedView(ConnectorSession session, SchemaTableName vie
                 .withStorage(storage -> storage.setStorageFormat(VIEW_STORAGE_FORMAT))
                 .withStorage(storage -> storage.setLocation(""))
                 .setViewOriginalText(Optional.of(
-                        encodeMaterializedViewData(fromConnectorMaterializedViewDefinition(definition))))
+                        encodeMaterializedViewData(
```

Comment:
We keep storage table in Hive view properties (it was not encoded previously as part of `ConnectorMaterializedViewDefinition`). We also don't use MV properties, but rather table properties. Maybe let's encode just needed fields in `IcebergMaterializedViewDefinition`?

Reply:
I've removed storageTable from IcebergMaterializedViewDefinition now